### PR TITLE
Update sendmail fast-cgi parameters

### PR DESCRIPTION
**Pull request by ghost**
Ensure that outgoing mail originates from the hosted domain and not the server hostname. #573

Please review to ensure templates are correct and that no items were missed.

### CODE REVIEW

**Comment by Lupul** [comment]
Can we use php-fpm templates to set these flags like we have here? :
https://github.com/hestiacp/hestiacp/blob/master/install/deb/php-fpm/multiphp.tpl#L20
eg: /templates/web/php-fpm/default.tpl and /templates/web/php-fpm/socketdefault.tpl

File: install/deb/templates/web/nginx/hosting.tpl [hunk 1]
```diff
@@ -14,7 +14,12 @@ server {
             try_files      $uri @fallback;
         }
     }
-
+    
+    location ~ [^/]\.php(/|$) {
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+        fastcgi_param PHP_ADMIN_VALUE "sendmail_path = /usr/sbin/sendmail -t -i -f admin@%domain%";
```
Comment (skullwritter) [inline install/deb/templates/web/nginx/hosting.tpl]: i was wandering about the following code:
```
if (-f /usr/sbin/sendmail){

        fastcgi_param PHP_ADMIN_VALUE "sendmail_path = /usr/sbin/sendmail -t -i -f admin@%domain%";
}
```
cause the system might not have the sendmail in that path somehow or not installed

**Review by tlcd96 — COMMENTED** [review]
i was wandering about the following code:
```
if (-f /usr/sbin/sendmail){

        fastcgi_param PHP_ADMIN_VALUE "sendmail_path = /usr/sbin/sendmail -t -i -f admin@%domain%";
}
```
cause the system might not have the sendmail in that path somehow or not installed

**Comment by ghost** [comment]
Updated globally in php-fpm config, thanks @Lupul!

**Pull request closed without merging**